websockify/token_plugins.py

```diff
@@ -79,4 +79,4 @@ class JSONTokenApi(BaseTokenAPI):
     # should go
 
     def process_result(self, resp):
-        return (resp.json['host'], resp.json['port'])
+        return (str(resp.json()['host']), str(resp.json()['port']))
```

Comment:
On my test, I have error message:
handler exception: Int or String expected

When I change the line to:
return (str(resp.json['host']), str(resp.json['port']))

it solve the error. So on my case the str() is needed.
And the resp.json()[] is not work, so it should use just resp.json[].

 My python is version 2.6.6